components/SingleArticle.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { BodyHeader } from "./BodyHeader";
```

Comment:
This is causing an error when I'm trying to view as you have not using capital letters in the component file name